resources/core/charts/cluster-users/templates/rbac-roles.yaml

```diff
@@ -30,7 +30,7 @@ metadata:
   annotations:
     "helm.sh/hook-weight": "0"
 rules:
-- apiGroups: ["", "apps", "authentication.kyma-project.io", "extensions", "gateway.kyma-project.io", "kubeless.io", "rbac.authorization.k8s.io", "applicationconnector.kyma-project.io", "servicecatalog.k8s.io", "servicecatalog.kyma.cx", "settings.k8s.io", "kyma.cx", "authentication.istio.io", "config.istio.io", "eventing.kyma.cx", "ui.kyma.cx", "ui.kyma-project.io", "metrics.k8s.io"]
+- apiGroups: ["", "apps", "authentication.kyma-project.io", "extensions", "gateway.kyma-project.io", "kubeless.io", "rbac.authorization.k8s.io", "applicationconnector.kyma-project.io", "servicecatalog.k8s.io", "servicecatalog.kyma.cx", "settings.k8s.io", "kyma.cx", "authentication.istio.io", "config.istio.io", "eventing.kyma.cx", "ui.kyma.cx", "ui.kyma-project.io", "metrics.k8s.io", "virtualservices.networking.istio.io"]
```

Comment:
`virtualservices` is resource kind. `networking.istio.io` should be sufficient

Reply:
hmm, I like the idea of increasing permissions to a whole networinkg.istio.io group

Reply:
Done!